submarine-client/src/main/java/org/apache/submarine/client/cli/param/ParametersHolder.java

```diff
@@ -461,4 +461,32 @@ public Parameter setParameters(BaseParameters parameters) {
     this.parameters = parameters;
     return this;
   }
+
+  public CommandLine getParsedCommandLine() {
+    return parsedCommandLine;
+  }
+
+  public Parameter setparsedCommandLine(CommandLine parsedCommandLine) {
```

Comment:
Thanks for the patch @yuanzac 
`setparsedCommandLine` need to rename to `setParsedCommandLine`